app/backend/middleware/session.ts

```diff
@@ -0,0 +1,44 @@
+import crypto from 'crypto';
+import expressSession from 'express-session';
+import connectPgSimple from 'connect-pg-simple';
+import { pgPool } from '../db/setup-pg';
+
+const PgSession = connectPgSimple(expressSession);
+const sessionSecret =
+  process.env.SESSION_SECRET || crypto.randomBytes(32).toString();
+const isProd = process.env.NODE_ENV === 'production';
+
+const HALF_DAY = 12 * (60 * 60 * 1000);
+const ONE_DAY = 2 * HALF_DAY;
+const TWO_WEEKS = 14 * ONE_DAY;
+const THIRTY_DAYS = 30 * ONE_DAY;
+
+const session = () => {
+  const store = new PgSession({
+    pool: pgPool,
+    schemaName: 'connectivity_intake_private',
+    tableName: 'connect_session'
+  });
+
+  const middleware = expressSession({
+    secret: sessionSecret,
+    resave: false,
```

Comment:
Are we absolutely sure we do not want to resave the session? Let's add a card to validate this?